static/templates/user_profile_modal.hbs

```diff
@@ -85,6 +89,9 @@
                             <table class="user-group-list" data-empty="{{t 'No user group subscriptions.'}}"></table>
                         </div>
                     </div>
+                    {{#if can_manage_user}}
+                        <div class="tabcontent" id="user-profile-manage-user-tab"></div>
+                    {{/if}}
```

Comment:
Would it be more convenient for this `div` to always be present, and just be empty when `can_manage_user` is false?